internal/app/mistapiconnector/mistapiconnector_app.go

```diff
@@ -473,6 +473,77 @@ func (mc *mac) generateRouteKeys(stream *livepeer.CreateStreamResp) []string {
 	return keys
 }
 
+func (mc *mac) triggerUserNew(w http.ResponseWriter, r *http.Request, lines []string, rawRequest string) bool {
+	if len(lines) != 6 {
+		glog.Errorf("Expected 6 lines, got %d, request \n%s", len(lines), rawRequest)
+		w.WriteHeader(http.StatusBadRequest)
+		w.Write([]byte("false"))
+		return false
+	}
+
+	playbackId := lines[0]
+	stream, err := mc.lapi.GetStreamByPlaybackID(playbackId)
+	if err != nil || stream == nil {
+		glog.Errorf("Error getting stream info from Livepeer API playbackId=%s err=%v", playbackId, err)
+		w.WriteHeader(http.StatusNotFound)
+		w.Write([]byte("false"))
+		return false
+	}
+
+	userId := stream.UserID
+	userWebhooks, err := mc.lapi.GetWebhooksByUserId(userId, "user.new")
+	if err != nil {
+		glog.Errorf("Error getting webhooks for user %s err=%v", userId, err)
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte("false"))
+		return false
+	}
```

Comment:
Same as above, I think we can even avoid calling this function every time a new user starts playing as well. Instead, we could call `GetWebhooksByUserId` as soon as the stream starts and save it into the `streamInfo` struct as well. 

i.e. Would add a `userNewWebhooks` field in `streamInfo` struct and initialize it with the response from this API when we create a stream info [here](https://github.com/livepeer/stream-tester/blob/fce88570dc0c3e7f86c86e4a8076e996b58d0880/internal/app/mistapiconnector/mistapiconnector_app.go#L546).

That way we avoid **all** I/O while handling this hook apart from the inevitable ones (calling the webhooks themselves). Since we are having lots of network issues in our clusters could be a good idea. WDYT?